site/static/css/styles.css

```
@@ -2371,11 +2375,20 @@ h4,
 h5,
 h6 {
   font-weight: 400;
-  margin-bottom: 1em; }
+  margin-bottom: 1em;
+  margin-top: 1.8em; }
```

nit: locate bracket on next line?
```suggestion
  margin-top: 1.8em;
}
```